modules/coreboot

```diff
@@ -137,11 +135,17 @@ else
 endif
 
 $(coreboot_module)_configure := \
-	mkdir -p "$(build)/$(coreboot_dir)" \
-	&& $(call install_config,$(pwd)/$(CONFIG_COREBOOT_CONFIG),$(build)/$(coreboot_dir)/.config) \
-	&& echo 'CONFIG_LOCALVERSION="$(BRAND_NAME)-$(HEADS_GIT_VERSION)"' >> $(build)/$(coreboot_dir)/.config \
-	&& echo 'CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="$(BOARD)"' >> $(build)/$(coreboot_dir)/.config \
-	&& $(MAKE) olddefconfig \
+	mkdir -p "$(build)/$(coreboot_dir)"; \
+	$(call install_config,$(pwd)/$(CONFIG_COREBOOT_CONFIG),$(build)/$(coreboot_dir)/.config); \
+	sed -i '/^CONFIG_LOCALVERSION/d' $(build)/$(coreboot_dir)/.config; \
+	echo 'CONFIG_LOCALVERSION=$(CONFIG_COREBOOT_LOCALVERSION)' >> $(build)/$(coreboot_dir)/.config; \
+	sed -i '/^CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME/d' $(build)/$(coreboot_dir)/.config; \
+	echo 'CONFIG_MAINBOARD_SMBIOS_PRODUCT_NAME="$(CONFIG_COREBOOT_SMBIOS_PRODUCT_NAME)"' >> $(build)/$(coreboot_dir)/.config; \
+	if [ ! -z "$(CONFIG_COREBOOT_SMBIOS_MANUFACTURER)" ]; then \
+		sed -i '/^CONFIG_MAINBOARD_SMBIOS_MANUFACTURER/d' $(build)/$(coreboot_dir)/.config; \
+		echo 'CONFIG_MAINBOARD_SMBIOS_MANUFACTURER="$(CONFIG_COREBOOT_SMBIOS_MANUFACTURER)"' >> $(build)/$(coreboot_dir)/.config; \
+	fi; \
```

Comment:
I don't think we need  `CONFIG_COREBOOT_SMBIOS_MANUFACTURER` or `CONFIG_COREBOOT_SMBIOS_PRODUCT_NAME`.  Can we remove them?

The Nitropad coreboot configs appear to already have the appropriate values (they're using Clevo coreboot ports but the BIOS will say Nitropad, etc.), which IMO is the right way to do it anyway.

I would prefer to stop overriding the board name anyway - with it, the product name for a device differs between SeaBIOS and Heads, which causes problems in udev rules, etc. (e.g. `Librem 14` in SeaBIOS vs. `librem_14` in Heads).  I suspect it is similar for other boards but am open to discussing any other needs :slightly_smiling_face: 

@tlaurion How do you feel about stopping those substitutions entirely?  If you'd prefer not to do that in this Nitrokey PR we could do it separately, though I would suggest at least not extending the mechanism in this PR (remove the new CONFIG_COREBOOT_SMBIOS_* vars here at least, IMO).

Reply:
Oh I should have read the whole PR, sorry, I see some comments above about site-local, let me look over that